plugins/org/src/components/Cards/User/UserProfileCard/UserProfileCard.tsx

```diff
@@ -80,40 +89,35 @@ export const UserProfileCard = ({
     return <Alert severity="error">User not found</Alert>;
   }
 
+  const emailHref = profile?.email ? `mailto:${profile.email}` : '';
+
   return (
     <InfoCard title={<CardTitle title={displayName} />} variant={variant}>
-      <Grid container spacing={3}>
-        <Grid item xs={12} sm={2} xl={1}>
-          <Box
-            display="flex"
-            alignItems="flex-start"
-            justifyContent="center"
-            height="100%"
-            width="100%"
-          >
-            <Avatar displayName={displayName} picture={profile?.picture} />
-          </Box>
+      <Grid container spacing={3} alignItems="flex-start">
+        <Grid item md={2} lg={1}>
+          <Avatar displayName={displayName} picture={profile?.picture} />
         </Grid>
-        <Grid item md={10} xl={11}>
-          {profile?.email && (
-            <Typography variant="subtitle1">
-              <Box display="flex" alignItems="center">
-                <Tooltip title="Email">
-                  <EmailIcon fontSize="inherit" />
-                </Tooltip>
 
-                <Box ml={1} display="inline">
-                  {profile.email}
-                </Box>
-              </Box>
-            </Typography>
-          )}
-          <Typography variant="subtitle1">
-            <Box display="flex" alignItems="center">
-              <Tooltip title="Member of">
-                <GroupIcon />
-              </Tooltip>
-              <Box ml={1} display="inline">
+        <Grid item md={10} lg={11}>
+          <List>
+            {profile?.email && (
+              <ListItem>
+                <ListItemIcon>
+                  <EmailIcon />
+                </ListItemIcon>
+                <ListItemText>
+                  <Link href={emailHref}>{profile.email}</Link>
```

Comment:
This code isn't using `Link` from `@backstage/core`, but I'm not sure if that one would support `mailto:`

Reply:
If it doesn't we should make it!

Reply:
* @Fox32 , yes this works.  This is the Material-UI Link, which allows href, and this is just translated back to standard href links via HTML.
* @freben agree.  Fox32 is correct - `@backstage/core` requires the variable `to` and thus always needs a route, and does NOT support providing href's i.e. URLs.

Reply:
It might be a good idea to check if email is set before displaying the mailto link. I'm pretty sure this is field is optional and in the case of GitHub users there are many times people do not make it public.

Reply:
@andrewthauer Should already be handled because that entire section is wrapped with a ~{ profile?.email ? }` element. So if there's no e-mail, the list item won't even display. (I checked on a number of my users who I know lacked e-mails also.)